wled00/data/settings_pin.htm

```diff
@@ -11,11 +11,20 @@
 	<style>
 		@import url("style.css");
 	</style>
+	<script>
+		function checkNum(o) {
+			const specialkeys = ["Backspace", "Tab", "Enter", "Shift", "Control", "Alt", "Pause", "CapsLock", "Escape", "Space", "PageUp", "PageDown", "End", "Home", "ArrowLeft", "ArrowUp", "ArrowRight", "ArrowDown", "Insert", "Delete"];
```

Comment:
what does this line do and is it necessary or more of a best practice? it adds quite a lot of code...